test_sets/policies_smoke.txt

```diff
@@ -0,0 +1,6 @@
+./test_scripts/Polices/build_options/001_ATF_P_TC_SDL_Build_Flag_DEXTENDED_POLICY_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_kilometers.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_days.lua
+./test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua
+./test_scripts/Polices/build_options/ATF_P_TC_PTU_Transfer_Several_Apps_Different_HMI_Levels_PROPRIETARY.lua
+./test_scripts/Polices/build_options/ATF_P_PTS_creation_rule_PROPRIETARY.lua
```

Comment:
@dboltovskyi, ACCEPTED
Suggestion: you can use only of the scripts, because they have common verification.
Just for information: "But ATF_Start_PTU_retry_sequence.lua is not available"
https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/feature/external_proprietary_policy/test_scripts/Polices/build_options/ATF_Start_PTU_retry_sequence.lua